### ISSUE TITLE
Persistent d3dx Edits after XXMI startup 

### ISSUE THREAD

**Issue by raphyand** [SpectrumQT/WWMI#21]
Hello again, I made an [issue](https://github.com/SpectrumQT/WWMI/issues/7) here before, and from that point, I was able to to do a workaround with this by having the log statements in the d3dx:

```
; Log all API usage
calls = 1

; Super verbose massive log
debug = 0

; Unbuffered logging to avoid missing anything at file end
unbuffered = 1
```
However, with the new Installer, it looks like it overwrites the d3dx file when I launch the game/edits don't persist,  and causes

```
; Super verbose massive log
debug = 1
```
which causes my game to become super laggy.

Is there a way to prevent overwriting with the Installer? Or to just keep verbose as 0?

**Comment by SpectrumQT** [comment]
Disable **Debug Logging** in [XX]MI section of launcher Settings to make them zeroes. Edit **d3dx_ini** dict in `XXMI Launcher Config.json` if you want some other combination for on/off defaults.

**Comment by raphyand** [comment]
Gotcha - I did end up trying it, and it strangely worked for one load, then it seems the `config.json` doesn't adjust `d3dx.ini` file.  For some reason I couldn't replicate the successful run.

I tried looking through the XXMI source code but couldn't quite narrow down where the logic happens; 

For instance, when adjusting in the XXMI launcher settings, it does adjust the `config.json`, but when actually editing the config, it doesn't change the field in the `d3dx.ini`.  

Does adjusting the config work on your end with the WWMI Dict, and reflect in the `.ini` ?  Just troubleshooting if my system just has some weird stuff going on.
